src/server.rs

```diff
@@ -93,6 +93,14 @@ pub fn new() -> ServerPtr {
     Arc::new(RwLock::new(server))
 }
 
+fn mac_wakeup(){
+    #[cfg(target_os = "macos")]{
+        use std::process::Command;
+        Command::new("/usr/bin/caffeinate").arg("-u").arg("-t 5").spawn().expect("failed to execute caffeinate");
```

Comment:
expect may panic. please remove it. just use ok();
https://doc.rust-lang.org/std/result/enum.Result.html#method.expect

Reply:
Thanks